Updated OS package repo docs (#31541)
gravitational/teleport: docs/pages/management/operations/self-hosted-automatic-agent-updates.mdx

Systemd agents enrolled into automatic updates can only install versions
present in their package repositories. As Teleport 14 won't be published to
`stable/v13`, those agents will require manual intervention to be updated to
the next major version (adding a new APT/YUM/zypper repo for `stable/v14`).

> Alternatively, the `stable/rolling` channel can be used which contains

Reviewer: ```suggestion
Alternatively, you can use the `stable/rolling` channel, which contains
```

Author: This changes the text to a second person perspective. Is that intended? I think most of our docs are third person, but may be mistaken.

Reviewer: It's pretty common in our docs (and docs in general) to address the reader. I made this change to use active voice instead of passive.

> Alternatively, the `stable/rolling` channel can be used which contains

Reviewer 2: The way we mention `stable/rolling` channel feels very "and by the way you can use this channel" which I think might be easy to miss unless you actually read the entire guide carefully.

I wonder if it would be useful to have a table somewhere on Installation page that describes what channels we have and what gets published to them. Then users can decide which one they want to use. @fheinecke @ptgott What do you guys think?

Something like:

```
Channel name    | Description
---------------------------
stable/<major>  | Receives releases for a particular major release line
stable/cloud    | Rolling channel that receives releases compatible with current Cloud version
stable/rolling  | Rolling channel that receives all published Teleport releases
```

Reviewer: That sounds good to me!
